src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -981,6 +998,108 @@ public sealed class BottleneckMetrics
             public override string ToString() => $"Phase: Bottleneck Computation, Dataset used: {DatasetUsed.ToString(),10}, Image Index: {Index,3}, Image Name: {Name}";
         }
 
+        /// <summary>
+        /// Early Stopping feature stops training when monitored quantity stops improving'.
+        /// Modeled after https://github.com/tensorflow/tensorflow/blob/00fad90125b18b80fe054de1055770cfb8fe4ba3/tensorflow/python/keras/callbacks.py#L1143
+        /// </summary>
+        public sealed class EarlyStopping
+        {
+            /// <summary>
+            /// Best value of metric seen so far.
+            /// </summary>
+            private float _bestMetricValue;
+
+            /// <summary>
+            /// Current number of epochs where there has been no improvement.
+            /// Stop training when wait >=patience.
```

Comment:
>wait [](start = 35, length = 4)

please use param ref to refer variables.

Reply:
updated description to not use variables. Couldn't use <paramref>  as these variables are not parameters to this particular variable(i guess).

---
In reply to: [330316879](https://github.com/dotnet/machinelearning/pull/4237#discussion_r330316879) [](ancestors = 330316879)